examples/knative/python/kn-py-slack/README.md

```diff
@@ -0,0 +1,117 @@
+# kn-ps-slack
+Example Knative Python function for sending to a Slack webhook when a Virtual Machine is powered off.
+
+# Step 1 - Build with `pack`
+
+[Buildpacks](https://buildpacks.io) are used to create the container image.
+
+```bash
+IMAGE=<docker-username>/kn-py-slack:1.0
+pack build -B gcr.io/buildpacks/builder:v1 ${IMAGE}
+```
+
+
+# Step 2 - Test
+
+Verify the container image works by executing it locally.
+
+Change into the `test` directory
+```console
+cd test
+```
+
+Update the `docker-test-env-variable` file with your Slack webook URL.
+
+Start the container image by running the following command:
+
+```console
+docker run -e FUNCTION_DEBUG=true -e PORT=8080 --env-file docker-test-env-variable -it --rm -p 8080:8080 <docker-username>/kn-py-slack:1.0
+
+Serving Flask app "handler.py" (lazy loading)
+ * Environment: development
+ * Debug mode: on
+ * Running on all addresses.
+   WARNING: This is a development server. Do not use it in a production deployment.
+ * Running on http://172.17.0.2:8080/ (Press CTRL+C to quit)
+ * Restarting with stat
+ * Debugger is active!
+ ```
+ 
+In a separate terminal window, go to the test directory and use the ```testevent.json``` file to validate the function is working. You should see output similar to this below.
```

Comment:
```suggestion
In a separate terminal window, go to the test directory and use the `testevent.json` file to validate the function is working. You should see output similar to this below.
```

Reply:
made the change and signed commit 9d9afc8.